runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpBindingConfig.java

```diff
@@ -49,9 +69,19 @@ public final class HttpBindingConfig
     public final List<HttpRouteConfig> routes;
     public final ToLongFunction<String> resolveId;
     public final Function<Function<String, String>, String> credentials;
+    public final List<HttpRequestType> requests;
+    public Map<String, String8FW> pathParams;
+    public Map<String, String8FW> queryParams;
```

Comment:
All field on binding should be final with no stateful side effects of calling methods like `resolveRequest`.